package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "main": "node_modules/expo/AppEntry.js",
```

Comment:
Seems like we're missing a few key fields here such as name, version, description, author, license (MIT), repository, homepage, bugs - you can mostly copy those from the smart contracts repository.

Reply:
ok